content/blog-posts/2023-08-16-ebpf-for-security/index.md

```diff
@@ -0,0 +1,10 @@
+---
+
+date: '2023-08-16T11:36:00.000Z'
+title: 'EBPF For Security: Evolution Or Revolution?'
+ogImage: ogimage.jpeg
+description: 'Explore eBPF rise as a pivotal tool for Linux kernel security. Learn about its transition from a network packet filter to a powerful tool for tracing and analysing system activity, and its transformative role in modern cloud and internet security'
```

Comment:
```suggestion
description: 'Explore eBPF's rise as a pivotal tool for Linux kernel security. Learn about its transition from a network packet filter to a powerful tool for tracing and analysing system activity, and its transformative role in modern cloud and internet security'
```

Reply:
we can't add the apostrophe because it'll break the build. That's why I left it that way, what I can do is to rephrase to 
"Explore the rise of ebPF"

Reply:
If you switch from ' to " it won't break the build

Reply:
oh, didn't know this